README.md

```diff
@@ -1,2 +1,3 @@
 # fiware-health
 Program to check the functionality of each FIWARE Lab node.
```

Comment:
Maybe, we merge the scripts related to bluerpint with this one, but not put different scripts. Just unify in only one place all the different code.